include/itkBackscatterImageFilter.h

```diff
@@ -0,0 +1,150 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkBackscatterImageFilter_h
+#define itkBackscatterImageFilter_h
+
+#include <vector>
+
+#include "itkImage.h"
+#include "itkImageToImageFilter.h"
+#include "itkMacro.h"
+#include "itkNumericTraits.h"
+
+namespace itk
+{
+/** \class BackscatterImageFilter
+ * \brief Computes the estimated backscatter coefficient
+ *
+ * BackscatterImageFilter receives an input vector image representing
+ * RF spectra. One image direction represents the direction of an
+ * RF waveform emitted from an ultrasound probe. Remaining image
+ * directions may represent directions in physical space, such as
+ * where the probe contains multiple elements, or directions in
+ * time as multiple samples are captured in a sweep with the probe.
+ * Each pixel in the input image is a vector representing frequency
+ * components at bins based on the sampling frequency.
+ *
+ * BackscatterImageFilter generates a scalar output image with
```

Comment:
Update to describe each of the three output images

Reply:
The second force-push did that? Lines 45-50.

Reply:
It would be good to clarify that the filter has three outputs. I.e.,

`BackscatterImageFilter generates a scalar output image ...` is no longer correct and should be changed to

`BackscatterImageFilter generates three scalar metric output images ...`